### ISSUE TITLE
Aperio xml files

### ISSUE THREAD

**Issue by Tato14** [sdw95927/Comprehensive-analysis-of-lung-cancer-pathology-images#1]
Hi!

I would like to know if you could share the aperio xml region annotation files from TCGA-LUAD database. Thanks!

**Comment by sdw95927** [comment]
Hi, 

sorry that our pathologists who created the labeling do not want to share the xml file. I believe you can easily create them through ImageScope.

Thanks!

**Comment by Tato14** [comment]
Hi! Thanks for the reply. Sadly, I do not have the possibility to ask any pathologist and do not have the knowledge to label the regions myself. 
In this case, could you share the weights of the CNN once trained? Did I missed this file somewhere? Thanks!

**Comment by sdw95927** [comment]
The model file is too big (~2GB) to be uploaded to github... If you can find a way to share such a big file, I'm willing to share the weights to you. Sorry for the inconvenience for now.

**Comment by Tato14** [comment]
Hi! In the [Keras documentation](https://keras.io/models/about-keras-models/) I see that you can export only the weights using `model.get_weights()`. Is this file smaller?

Thanks!

**Comment by sdw95927** [comment]
It's still as large as 1GB.. I found the reason is that we used dense layer instead of global pooling layer for the last 4th layer, which creates a big amount of variables. Will improve that in the future.

**Comment by sdw95927** [comment]
The weight file is now downloadable at https://drive.google.com/file/d/1zdrLQXypAd-XI1KQ0KOdjMqkfrxvQU7F/view?usp=sharing.
